gcd/src/main/java/org/spine3/server/storage/datastore/type/DsColumnTypes.java

```diff
@@ -0,0 +1,181 @@
+/*
+ * Copyright 2017, TeamDev Ltd. All rights reserved.
+ *
+ * Redistribution and use in source and/or binary forms, with or without
+ * modification, must retain the above copyright notice and the following
+ * disclaimer.
+ *
+ * THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS
+ * "AS IS" AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT
+ * LIMITED TO, THE IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR
+ * A PARTICULAR PURPOSE ARE DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT
+ * OWNER OR CONTRIBUTORS BE LIABLE FOR ANY DIRECT, INDIRECT, INCIDENTAL,
+ * SPECIAL, EXEMPLARY, OR CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT
+ * LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES; LOSS OF USE,
+ * DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND ON ANY
+ * THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT
+ * (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE
+ * OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+ */
+
+package org.spine3.server.storage.datastore.type;
+
+import com.google.cloud.datastore.BaseEntity;
+import com.google.cloud.datastore.DateTime;
+import com.google.protobuf.AbstractMessage;
+import com.google.protobuf.Timestamp;
+import org.spine3.annotations.Internal;
+import org.spine3.base.Version;
+import org.spine3.json.Json;
+import org.spine3.protobuf.Timestamps2;
+
+import java.util.Date;
+
+/**
+ * A utility for creating the basic {@link DatastoreColumnType} implementations for
+ * <ul>
+ *     <li>{@code String}
+ *     <li>{@code Integer}
+ *     <li>{@code Boolean}
+ *     <li>{@link Timestamp}
+ *     <li>{@link Version}
+ *     <li>{@link AbstractMessage}
+ * </ul>
+ *
+ * @author Dmytro Dashenkov
+ */
+@Internal
+class DsColumnTypes {
```

Comment:
We agreed not to have `@Internal` annotation for the package-private classes. Please remove.